person.rb

```diff
@@ -31,16 +31,6 @@ def add_rentals(book, date)
   def correct_name
     @name
   end
-
-  def can_use_services?
-    @parent_permission || of_age?
-  end
-
-  private
-
-  def of_age?
-    @age >= 18
-  end
 end
 
 person = Person.new('maximilianus', 22)
```

Comment:
- [ ] Kindly remove the view of the person class which was to test the functionality of a previous milestone. 👏 

### View:
![image](https://github.com/IvonneBenitesRodriguez/oop-school-library/assets/50990925/dc602cf1-ac1d-45d2-a17e-a0d84993254d)

Reply:
Dear @Shedrack-Sunday,
thank you for your suggestions, and kind comments,
we appreciate it. We have modified it as you requested, 
thanks again, 
Best wishes